custom/icds_reports/tests/reports/test_adhaar.py

```diff
@@ -199,18 +199,18 @@ def test_chart_data(self):
             {
                 "location_type": "State",
                 "bottom_five": [
-                    {'loc_name': u'st4', 'percent': 0.0},
-                    {'loc_name': u'st5', 'percent': 0.0},
-                    {'loc_name': u'st6', 'percent': 0.0},
-                    {'loc_name': u'st7', 'percent': 0.0},
-                    {'loc_name': u'st3', 'percent': 0.0}
+                    {'loc_name': 'st4', 'percent': 0.0},
```

Comment:
@lwyszomi Now that all files have ```from __future__ import unicode_literals``` we don't need or want to prefix unicode with u.